.github/workflows/jira-transition.yml

```diff
@@ -17,8 +17,8 @@ on:
 
 jobs:
   call-label-action:
-    uses: dbt-labs/jira-actions/.github/workflows/jira-transition.yml@main
+    uses: dbt-labs/actions/.github/workflows/jira-transition.yml@main
```

Comment:
Similar comment, convert to [jira-transition-actions.yml](https://github.com/dbt-labs/actions/blob/main/.github/workflows/jira-transition-actions.yml)